packages/dev/core/src/Engines/engine.ts

```diff
@@ -298,7 +298,7 @@ export class Engine extends ThinEngine {
      * @param options An object that sets options for the image's extraction.
      * @returns ImageBitmap.
      */
-    public createImageBitmapFromSource(imageSource: string, options?: ImageBitmapOptions): Promise<ImageBitmap> {
+    public _createImageBitmapFromSource(imageSource: string, options?: ImageBitmapOptions): Promise<ImageBitmap> {

```

Comment:
This breaks backwards compatibility. Any reason for the change in the function name?

Reply:
Is this actually intoducing a Backwards compat issue, the only reference to this is in WebXRImage, which is also shipped in the @babylonjs\core package so it is only changing the interface for internal use.

I only updated this to include the underscore to match the naming pattern for other hidden methods, so happy to change it back if this does break backwards compat.

Reply:
discussed internally. resolving :-)